### PR TITLE
improve clauses-and-options

### PR DESCRIPTION
**Pull request by whitewum**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md [hunk 1]
```diff
@@ -1,10 +1,48 @@
 # GROUP BY
 
+## OpenCypher Compatibility
+
+This page applies to nGQL extensions only.
+
+Use `GROUP BY` in nGQL-extensions **ONLY** to aggregate data.
+
+The OpenCypher uses [count() function](../6.functions-and-expressions/7.count.md) to aggregate data.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md:9]: openCypher

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 2]
```diff
@@ -1,29 +1,68 @@
...
 
-## Syntax
+## OpenCypher Compatibility
 
-```nGQL
-LIMIT [<offset>,] <number_rows>
+The Syntax in openCypher and nGQL-extension are different.
+
+- nGQL-extension: A pipe `|` must be used. And an offset can be ignored.
+- openCypher: No pipe are permitted. Use `Skip` to indicate offset.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:10]: pipes

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 3]
```diff
@@ -1,29 +1,68 @@
...
 
 |Parameter|Description|
 |-|-|
-|`offset`|Optional. It specifies the number of rows to be skipped. The offset starts from zero.|
-|`number_rows`|It specifies the number of rows to be returned. It can be a non-negative integer or an expression that outputs a non-negative integer.|
+|`offset`| Optional. It specifies the number of rows to be skipped. The offset starts from zero.|
+|`number_rows`| It specifies the number of rows to be returned. It can be a non-negative integer or an expression that outputs a non-negative integer.|
 
-> **NOTE:** In a `MATCH` statement, you can use a `LIMIT` clause with or without a pipe symbol, but in other statements, you must add a pipe symbol before the `LIMIT` clause.
+Both parameters accept an expression, which must output a non-negative integer.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:55]: what are both parameters? Pls make it clear.

**Review by Amber1990Zhang — DISMISSED** [review]

**Review by randomJoe211 — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md [hunk 4]
```diff
@@ -1,29 +1,68 @@
-# LIMIT
+# LIMIT AND SKIP
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md:1]: Good idea!

**Review by randomJoe211 — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**